docker/koordlet.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$TARGETPLATFORM golang:1.18 as builder
+FROM --platform=$TARGETPLATFORM golang:1.20 as builder
```

Comment:
Because the koordlet depends on the glibc (feature Libpfm imported) and the golang:1.20 uses glibc >= 2.34, the Ubuntu image should be upgraded.